site/docs/components/switch/usage.mdx

```diff
@@ -0,0 +1,51 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Using the Switch component
+
+#### When to use Switch
+
+- To present an instantaneous, binary choice between two options that doesn’t require the action to be submitted or confirmed. The corresponding action takes effect immediately.
+- To control a critical action that may require confirmation from the user before applying the action. This delay is usually intentional, ensuring that users do not accidentally make unwanted changes.
+
+#### When not to use Switch
+
+- To present a list of independent options where the user can select any number of choices. Instead, use [Checkbox](../checkbox).
+- To make a single selection between mutually exclusive choices between two or more options. Instead, use [Radio Button](../radio-button).
+- To toggle between two (or more) opposing yet mutually exclusive states or options with visual priority. Instead, use [Toggle Button](../toggle-button).
+- To present multiple options in a group within a compact UI, or if they’re subject to change depending on context. Instead, use [Selectable Pill](../pill).
+
+### Import
+
+{/* Update the text and code snippet below as needed: */}
+
+To import Switch from the lab Salt package, use:
+
+```js
+import { Switch } from "@salt-ds/lab";
+```
+
+### Content
+
+- The option that that switch controls should be made clear in the text inline to the switch.
+- Text should always sit to the right of the switch.
+- Keep the text description as clear and concise as possible, and ensure it accurately describes the action the switch will perform upon interaction.
+
+### Props
+
+{/* Update packageName and componentName below as needed */}
+{/* packageName is optional and defaults to "core" if omitted */}
+
+<PropsTable packageName="lab" componentName="Switch" />
+
+### References
+
+- Switch Pattern (https://www.w3.org/WAI/ARIA/apg/patterns/switch/ ) W3C
```

Comment:
```suggestion
- [Switch Pattern W3C](https://www.w3.org/WAI/ARIA/apg/patterns/switch/)
```

Reply:
i think design wanted it displayed this way as is the case for a few other components, will leave it as is for now